lib/stripe/payout.rb

```diff
@@ -10,13 +10,9 @@ class Payout < APIResource
 
     custom_method :cancel, http_verb: :post
 
-    def cancel
-      resp, api_key = request(:post, cancel_url)
-      initialize_from(resp.data, api_key)
-    end
-
-    def cancel_url
-      resource_url + "/cancel"
+    def cancel(params = {}, opts = {})
+      resp, opts = request(:post, resource_url + "/cancel", params, opts)
+      initialize_from(resp.data, opts)
     end
```

Comment:
`cancel_url` method was technically public. I don't see it being used for anything outside this, but still, this is a public API change. Happy to revert this if we're not comfortable with it.

Reply:
we can stop using it ourselves, but agreed that we should not make a backwards-incompatible change, so let's leave the definition of `cancel_url` for now.

Reply:
Fixed

Reply:
Can you just add a comment to mark the method as deprecated, so that we don't forget to remove it in the next major version?

Reply:
+1 what @ob-stripe said about a comment.

It's also generally super easy to add a deprecation warning for this type of stuff, so consider doing that. Example of how here:

https://github.com/stripe/stripe-ruby/blob/master/lib/stripe/source.rb#L27-L28